scalableminds/webknossos
# [WIP] Use named rather than positional SQL queries

Making the code independent of the column order in the database. This means, we can keep the schema updated (rather than appending SQL evolutions)

### Steps to test:
- branch from wk/master, deploy that on dev, create some stuff
- then merge this branch into that new one, upgrade it on dev (without updating the schema there)
- should still work

### Issues:
- fixes #2468 

------
